cmd/plume/README.md

```diff
@@ -24,6 +139,25 @@ bin/plume pre-release -C user --verify-key ~/keyfile -V $version-$COREOS_BUILD_I
 bin/plume release -C user -V <version>-$COREOS_BUILD_ID
 ```
 
+### Fedora Cloud Releases
+
+There are two sub-commands to do a Fedora Cloud Release: `pre-release` and `release`. When using `pre-release` plume will download the compose-id from the specified channel, extract the contents, and upload it to S3. From there it will use the import VM feature within AWS to create a private AMI from S3 and it will make it available across all regions. When using `release` it will make these AMIs public.
+
+Here is an example of doing a Fedora Cloud pre-release with plume:
+
+```
+./bin/plume pre-release \
```

Comment:
do you have an example running the `release` step? I'm getting an error and I don't know if I'm doing something dumb.

Reply:
This is what I found to work (note that I had to use `--distro` and not `--system`: 

```
plume release --distro fedora --channel cloud --version 30 --timestamp 20190925 --respin 0 --board x86_64 --compose-id Fedora-Cloud-30-20190925.0 --image-type Cloud-Base --debug
```

Though that led to other errors.

Reply:
What exact errors? I never did the release because I didn't want to make a bunch of AMIs public on my AWS account.

Reply:
i'm working through them now